packages/x-client/src/utils/crypto/crypto.ts

```diff
@@ -4,7 +4,8 @@ import BN from 'bn.js';
 import elliptic from 'elliptic';
 import * as encUtils from 'enc-utils';
 import { Signer } from '@ethersproject/abstract-signer';
-import { solidityKeccak256 } from 'ethers/lib/utils';
+// eslint-disable-next-line import/extensions
+import { solidityKeccak256 } from 'ethers/lib/utils.js';
```

Comment:
Another solution is importing the whole utils instead:

```suggestion
import { utils } from 'ethers/lib/utils.js';
...
utils.solidityKeccak256(...)
```

which I don't personally recommend given it'll add the whole `libs` declaration into the transpiled code.